.drone.yml

```diff
@@ -161,7 +161,7 @@ steps:
       - cd integration_tests/library
       - make test
   - name: mixed-source-and-customizations
-    image: 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci:4.0.0
```

Comment:
If we need an R 4.0 image its just R 4.0.5, not 4.0.0

Reply:
I just tried this and it still didn't work, unfortunately.